Sample/ViewController.swift

```diff
@@ -9,12 +9,9 @@
 import UIKit
 
 class ViewController: UIViewController {
-
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
     }
-
-
 }
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/kmpnz/Sample/blob/db347568e1ddc4d261e8ffeb82565d58b3969a2b/Sample/ViewController.swift#L17"></span>Files should have a single trailing newline.<br />
<code>trailing_newline</code> <code>ViewController.swift:17</code></del></td>
    </tr>
  </tbody>
</table>